### PR TITLE
Change for hat stabilizer

### PR DESCRIPTION
**Pull request by SpaceLoveSs13**
##About The Pull Request
Captain's beret(blue&white) both can now be attached to the Hat Stabilizer Mod.

##How This Contributes To The Skyrat Roleplay Experience
Much like every other hats, berets should also be able to chosen for preference now on top of your hats.

##Changelog
🆑 SpaceLove
add: NT has finally losen up restrictions on Hat Stabilizer Module, allowing captain's beret on top of it as well.
/:cl:

178133482-76d5dbd4-1f66-4994-bfaa-84beeb5cb513
178133487-3bb3f42d-65b1-4566-84d0-252022553a8c
178133494-3ef5cb12-7f8d-402b-96d9-acb7a2453d33
178133498-8401629a-1bb1-48c8-b8be-a056b3133b05

P.S. Don't pay attention to my webediting a one line.

### CODE REVIEW

**Comment by EOBGames** [comment]
Did you intend to make this PR to TG or Skyrat? Because your PR body is suggesting Skyrat.

**Comment by Zonespace27** [comment]
Told them to take it upstream (skyrat-ss13/skyrat-tg#14837)

**Comment by SpaceLoveSs13** [comment]
> Did you intend to make this PR to TG or Skyrat? Because your PR body is suggesting Skyrat.

What zonespace said.

File: code/modules/mod/modules/modules_general.dm [hunk 1]
```diff
@@ -505,9 +505,8 @@
 			/obj/item/clothing/head/nursehat,
 			/obj/item/clothing/head/chefhat,
 			/obj/item/clothing/head/papersack,
-			)) - /obj/item/clothing/head/caphat/beret
-			//Need to subtract the beret because its annoying
-
+			/obj/item/clothing/head/caphat/beret,
```
Comment (ShizCalev) [inline code/modules/mod/modules/modules_general.dm:508]:
```suggestion
			/obj/item/clothing/head/caphat/beret
			))
```

**Pull request merged**